pennylane/debugging.py

```diff
@@ -186,6 +187,23 @@ def reset_active_dev(cls):
         """Reset the global active device variable to None."""
         cls.__active_dev = None
 
+    @classmethod
+    def _execute(cls, batch_tapes):
+        """Execute tape on the active device"""
+        dev = cls.get_active_device()
+
+        valid_batch, _ = (
+            (batch_tapes, None)
+            if not dev.wires
+            else qml.devices.preprocess.validate_device_wires(batch_tapes, wires=dev.wires)
+        )
```

Comment:
Is the underscore needed here?

Reply:
Yes, the `validate_device_wires` functions returns two things: 
- The validated batch of tapes 
- A Null processing program 

The null processing program doesn't do anything, so we don't need to keep track of it.